app/assets/src/components/views/compare/SamplesHeatmapView/samples_heatmap_view.scss

```diff
@@ -31,6 +34,23 @@
     }
   }
 
+  .filterToggleContainer {
+    margin: auto;
+    width: 47px;
+    display: block;
+    text-align: center;
+    position: relative;
+
+    .arrowIcon {
+      color: $dark-grey;
+      background-color: $white;
+      font-size: $font-size-h4;
+      padding: 0 18px;
+      border-radius: 0 0 4px 4px;
+      box-shadow: 0 2px 2px 0 rgba(34, 36, 38, 0.2);
```

Comment:
Could you use the box-shadows in colors.scss/elements.scss or add an additional constant? (but make sure the ones that are already there aren't good enough)